components/introduction/modules/ROOT/pages/index.adoc

```diff
@@ -33,6 +33,6 @@ You can use a variety of tooling to deploy to SKALE Chains, such as Hardhat, Tru
 
 == Other
 
-* xref:skaled::index.adoc[More about SKALE Chains]
+* xref:introduction::overview.adoc[More about SKALE Chains]
```

Comment:
This page cannot be found; see the URL in screenshot for page not existing or link being incorrect.

![Uploading image.png…]()

Reply:
fixed